<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecentFile</class>
 <widget class="QWidget" name="RecentFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>50</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>398</width>
    <height>50</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>398</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">#RecentFile
{
margin-left: 1px;
margin-right: 1px;
}

#wText
{
border-bottom: 2px solid rgb(241, 240, 240);
}

#pArrow
{
border: 0px;
border-bottom: 2px solid rgb(241, 240, 240);
}

#lFileType
{
padding-left: 10px;
border: 0px;
border-bottom: 2px solid rgb(241, 240, 240);
}

#lTime
{
color: rgb(102, 102, 102)
}

#lFileName
{
color: rgb(51, 51, 51)
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lFileType">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wText" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lFileName">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lTime">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>%1 hours ago</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pArrow">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/tray_share_ico.png</normaloff>:/images/tray_share_ico.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
